robyn/argument_parser.py

```diff
@@ -26,6 +26,12 @@ def __init__(self) -> None:
             default=False,
             help="Development mode. It restarts the server based on file changes.",
         )
+        parser.add_argument(
+            "--file",
+            dest="file_path",
+            default=None,
+            help="The file associated.",
+        )
```

Comment:
What does this do?

Reply:
@sansyrox hi, I have changed the implementation. PTAL :)